### ISSUE TITLE
Are the models overfitting?

### ISSUE THREAD

**Issue by Khurdhula-Harshavardhan** [Khurdhula-Harshavardhan/Pneumonia-Diagnosis-Using-AI#10]
### CNN-Vanilla-Overfitting:

The **_Author_** is aware that the models are **overfitting** with prolonged exposure to dataset (higher epochs.) this can be clearly observed from the **_unstable plot of Validation Accuracy of the CNN_** (imbalanced-vanilla-CNN). However, for research purposes the **_Author_** has decided to leave the models in the current state that they are in. Thank you.
